src/pages/App.tsx

```diff
@@ -77,16 +78,13 @@ const BodyWrapper = styled.div<{ isAboutpage?: boolean }>`
   flex-direction: column;
   width: 100%;
   align-items: center;
+  min-height: calc(100vh - 148px);
```

Comment:
how do you get this 148px?

Reply:
header + footer heigh bro 
header height = 86px
footer height = 62px